src/components/Dashboard/TodoList.vue

```diff
@@ -635,8 +635,8 @@ export default {
   },
 
   methods: {
-    ...mapActions(['setARFilingYear', 'setArMinDate', 'setArMaxDate', 'setNextARDate', 'setCurrentFilingStatus']),
-    ...mapGetters(['isInGoodStanding']),
```

Comment:
Getters don't belong here. This worked because this property was used as `this.isInGoodStanding()`. Note that this is a function call, while getters normally look like properties.